var/spack/repos/builtin/packages/py-jsonpath-ng/package.py

```diff
@@ -15,9 +15,11 @@ class PyJsonpathNg(PythonPackage):
     homepage = "https://github.com/h2non/jsonpath-ng"
     pypi = "jsonpath-ng/jsonpath-ng-1.5.2.tar.gz"
 
+    version("1.6.9", sha256="5483f8e9d74c39c9abfab554c070ae783c1c8cbadf5df60d561bc705ac68a07e")
```

Comment:
There is no `1.6.9` version, I think you mean `1.6.0` (which DOES have that sha256).